### ISSUE TITLE
Github IDs with Capital Letters

### ISSUE THREAD

**Issue by jkuruzovich** [jupyterhub/oauthenticator#52]
I’m using login with Github option and having some issues with a few students logging into Jupyterhub. I’ve just tested and I think it is because of capital letters. Unix doesn’t like the capital letters but i’ve tried it both ways (giving all lower case or forcing all upper case) and doesn't seem to allow either way.  

Is this option able to handle logging in with GitHub ID's that contain capital letters?

**Comment by minrk** [comment]
What version of JupyterHub and oauthenticator? The existing normalization should take care of this.

**Comment by jkuruzovich** [comment]
I'm using the https://github.com/jupyterhub/jupyterhub-deploy-teaching. 
    - jupyterhub==0.6
    - oauthenticator==0.3

I set to manually create the users: 
adduser -q --home /home/ilovedatascience8 --gecos "" ilovedatascience8 --disabled-password
Then I manually added them to the whitelist in the jupypter config file. 

The original userID is: ILoveDataSCIENCE8 
(I also tried using the capital version and it didn't work either. 

I'm unable to find anything else in the logs but it results in a 443 error.
